### PR TITLE
Fix build with `omake` 0.10.1

### PR DESCRIPTION
**Pull request by smondet**
Cf. https://github.com/ocaml-omake/omake/issues/72#issuecomment-265471550

### CODE REVIEW

File: OMakeroot [hunk 1]
```diff
@@ -13,7 +13,7 @@ DESCRIPTION = Workflow Engine For The Trenches
 
 if $(test -e .git)
     # this one works with quotes not the `None` one
```
Comment (ihodes) [inline OMakeroot]: This comment probably can be removed now?

Comment (smondet) [inline OMakeroot]: Ah yes :)

File: OMakeroot [hunk 2]
```diff
@@ -13,7 +13,7 @@ DESCRIPTION = Workflow Engine For The Trenches
...
-    GIT_COMMIT = 'Some "$(shell git rev-parse HEAD)"'
+    GIT_COMMIT = Some "$(shell git rev-parse HEAD)"
```
Comment (smondet) [inline OMakeroot]: Yes, see the comments in the omake issue.

Comment (ihodes) [inline OMakeroot]: Oh gotcha, sorry. Super weird!

**Comment by ihodes** [comment]
LGTM % dropping that comment

**Pull request merged**